### PR TITLE
CodeQuality: Removed logging for known exceptions

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

- https://files-org.sentry.io/issues/5512246531/events/3cd1ff17ebd34bbd81a5f3b2a8944c8f/
- https://files-org.sentry.io/issues/5515345913/events/573446f4b79d459c9d1db5e42b0fb029/
- https://files-org.sentry.io/issues/5833478165/events/ed325d08115440fe978f91e2a41a06bd/
- https://files-org.sentry.io/issues/5532296438/events/53aa0c6a83ea4364893350f8a4b6a623/
- https://files-org.sentry.io/issues/5833504869/events/a32230ee8261466e80aa8d9a52cd0d92/
- https://files-org.sentry.io/issues/5833462955/events/7398ae8ff92d42df9e82b343b75fad3c/
- https://files-org.sentry.io/issues/5833469890/events/8acdc13ae2ea4648af9c4ff4bacb27d9/
- https://files-org.sentry.io/issues/5833466595/events/d12e6d1f144540d29974c5db51d6d4f0/
- https://files-org.sentry.io/issues/5833460461/events/31707c0fb3574510bf47dd2e10c1b83f/
- https://files-org.sentry.io/issues/5833454351/events/3c54d6a210ca4073a5f330c6750a35dd/
- https://files-org.sentry.io/issues/5549328396/events/4921b660bffb4b7fa6ff22e44f5ea594/

**About**

The primary objective of this PR is to minimize unnecessary noise in Sentry. To achieve this, I’ve lowered the logging level for specific exceptions that do not require reporting to Sentry.

### CODE REVIEW

File: src/Files.App/Helpers/PathNormalization.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public static string NormalizePath(string path)
 			}
 			catch (Exception ex) when (ex is UriFormatException || ex is ArgumentException)
 			{
-				App.Logger.LogWarning(ex, path);
+				App.Logger.LogDebug(ex, path);
```
Comment (hishitetsu) [inline src/Files.App/Helpers/PathNormalization.cs:57]: How do you use LogInformation or LogDebug depending on the situation?

File: src/Files.App/Services/App/AppUpdateSideloadService.cs [hunk 2]
```diff
@@ -206,9 +210,17 @@ private async Task StartBackgroundDownloadAsync()
 
 				IsUpdateAvailable = true;
 			}
-			catch (Exception e)
+			catch (HttpIOException ex)
```
Comment (hishitetsu) [inline src/Files.App/Services/App/AppUpdateSideloadService.cs:213]: HttpIOException is a subclass of IOException, so this is not necessary.

File: src/Files.App/Helpers/PathNormalization.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Helpers/PathNormalization.cs:57]: LogInformation for non errors, and LogDebug for errors that don't need to be uploaded to Sentry, but are still nice to have locally.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**